include/flamegpu/gpu/CUDAEnsemble.h

```diff
@@ -7,6 +7,7 @@
 #include <vector>
 
 #include "flamegpu/defines.h"
+#include "flamegpu/io/Telemetry.h"
```

Comment:
This + constructor for `EnsembleConfig` should probably be moved to `CUDAEnsemble.cu` to prevent the include leaking into rest of the headers, It's not something we want users fiddling with.